content/post/articles/the-fish-shell-is-amazing.md

```diff
@@ -0,0 +1,95 @@
+---
+title: The fish shell is amazing
+date: 2021-10-8T00:15:28+01:00
+categories: [tutorial, blogging]
+authors: [RMPR]
+---
```

Comment:
There issue with the date, which causes the article to be displayed at the bottom of the list articles.

My guess is that this happened because the article was not generated with the hugo command
```
hugo new post/articles/my_new_post.md
/Users/<repository_path>/osscameroon-blog/content/post/articles/<my_new_post_name>.md created		
```

At least this is what happens when I run it locally. Do you mind having a look at it ?

<img width="336" alt="Screenshot 2021-11-08 at 11 34 15" src="https://user-images.githubusercontent.com/5704817/140727770-945aa690-1d3b-4b22-b40d-0034f5aff88a.png">
<img width="532" alt="Screenshot 2021-11-08 at 11 34 23" src="https://user-images.githubusercontent.com/5704817/140727773-af55d79b-4b99-407e-8f52-65705ae1747a.png">

Reply:
Sorry, I don't have hugo on this workstation yet, so I copy-pasted from my blog and added the date manually, should be good now.